modules/dxp/apps/commerce/commerce-organization-web/src/main/resources/META-INF/resources/js/OrganizationChart.js

```diff
@@ -42,7 +49,28 @@ function OrganizationChartApp({rootOrganizationId, spritemap, templatesURL}) {
 					zoomIn: zoomInRef.current,
 					zoomOut: zoomOutRef.current,
 				},
-				spritemap
+				spritemap,
+				{
+					open: (parentData, type) => {
```

Comment:
Can We extract these functions outside component closure? It will prevent us to have any callback side effect :)

You can create a function like `handleOrganizationChartOpen` and reuse it as 

```js
open: handleOrganizationChartOpen
```

Reply:
You can do the same for `close` and `open` callbacks

Reply:
Mh, not sure if I got the point but those functions make some updates to the component state so they cannot be moved outside. Also, since they're invoked just once, I preferred writing them inline

Reply:
> make some updates to the component state so they cannot be moved outside

ohhh ok! :)